.buildkite/pipeline.yml

```diff
@@ -0,0 +1,47 @@
+# Nodes with values to reuse in the pipeline.
+common_params:
+  plugins: &common_plugins
+  - &bash_cache automattic/bash-cache#v1.3.2: ~
+  # Common environment values to use with the `env` key.
+  env: &common_env
+    IMAGE_ID: xcode-12.5.1
+
+# This is the default pipeline – it will build and test the app
+steps:
+
+  #################
+  # Validate Podspec
+  #################
+  - label: "🔬 Validate Podspec"
+    key: "validate"
+    command: "validate_podspec"
+    env: *common_env
+    plugins: *common_plugins
```

Comment:
Why is `agents: { queue: "mac" }` required on the other steps but not here?

Reply:
Ah, good question! I think it works because it's the default value specified in the pipeline settings:

<img alt="image" src="https://user-images.githubusercontent.com/1218433/131338835-348efa73-9f37-4abf-baa4-c120e04cbc4c.png" width=300/>

Might be worth adding an explicit setting here, either in the step or a the top level, so that all the information to "resolve" the pipeline for a human reader is in the same file?

Reply:
Hmm yeah, the trade-off here is to specify the `queue` for each job entry independently. I'm ok with that, but I'd run around and change it everywhere separately I think?

Reply:
Yep I think it's worth it still, so the plan could be:
1. Update all existing `pipeline.yml` files at some point(†) to define the default `agent` at least at the top-level of the pipeline (like it's done [here](https://github.com/Automattic/ScreenObject/pull/6/files#r695336348))
2. Then remove the agent from the BuildKite UI's pipeline

(†) We might want to make a cleanup pass on all our pipelines anyway if we fancy also standardizing and unifying the way we declare our anchors too, so we could probably do both in one sweep 🙂